app/routes/settings+/profile.tsx

```diff
@@ -34,13 +34,13 @@ import {
 import { twoFAVerificationType } from './profile.two-factor.tsx'
 import { Icon } from '~/components/ui/icon.tsx'
 import { format } from 'date-fns'
-import { optionalDateSchema } from '~/utils/zod-extensions.ts'
+import { optionalDateTimeZoneSchema } from '~/utils/zod-extensions.ts'
 
 const profileFormSchema = z.object({
 	name: nameSchema.optional(),
 	username: usernameSchema,
 	email: emailSchema.optional(),
-	birthdate: optionalDateSchema,
+	birthdate: optionalDateTimeZoneSchema,
```

Comment:
Swapped this out to fix date creep bug. Every time user opens their profile to edit it, their birthday is a day earlier because without the time in a datetime object, midnight UTC is the day before AZ time.